apps/sharedtest/test/unit/telemetry_test.js

```diff
@@ -229,4 +229,28 @@ suite('Telemetry:', function() {
       });
     });
   });
+
+  /*
+   * Test that the getSettings() utility function works as expected.
+   * This is required for proper configuration of the module and for
+   * gathering the settings that are sent along with usage data.
+   */
+  suite('getSettings():', function() {
+    var getSettings;
+
+    suiteSetup(function() {
+      TelemetryRequest.DEBUG = false; // Shut up console output in test logs
+      getSettings = TelemetryRequest.getSettings;
```

Comment:
nit: looks like you're saving `getSettings` here, but don't use it below.. I'd say just get rid of it altogether :)